### ISSUE TITLE
compiling dicom-pixeldata with gdcm feature with gcc-11

### ISSUE THREAD

**Issue by yigitsoy** [Enet4/dicom-rs#221]
Thanks for developing  dicom-rs and dicom-pixeldata crates.

I need to support decompressing JPEG pixel data and and it seems I have to enable gdcm optional feature.

However, when  I enable it I face compilation issues like below

```
/home/ubuntu/.cargo/registry/src/github.com-1ecc6299db9ec823/gdcm-rs-0.2.0/GDCM/Source/MediaStorageAndFileFormat/gdcmImageChangePhotometricInterpretation.h:67:83: error: ‘::max’ has not been declared; did you mean ‘std::max’?
     67 |   return v < 0 ? 0 : (v > std::numeric_limits<T>::max() ? std::numeric_limits<T>::max() : v);
        |                                                                                   ^~~
        |                                                                                   std::max
```

Any advice how to proceed here?

Thanks!

**Comment by Enet4** [comment]
Thank you for reaching out, @yigitsoy. Yes, that is a known bug from GDCM, I have been using gcc-10 for the time being. I believe this has been fixed upstream, but the [gdcm-rs](https://github.com/pevers/gdcm-rs) crate needs to be updated accordingly, so that it uses a more recent revision of the library.

**Comment by pevers** [comment]
> Thank you for reaching out, @yigitsoy. Yes, that is a known bug from GDCM, I have been using gcc-10 for the time being. I believe this has been fixed upstream, but the [gdcm-rs](https://github.com/pevers/gdcm-rs) crate needs to be updated accordingly, so that it uses a more recent revision of the library.
> 
> CC @pevers If there is something we can do to help you drop a new release of gdcm-rs, please let us know.

I'll update the library and let you know when it is done!

**Comment by pevers** [comment]
@Enet4 I just updated `gdcm-rs` to 0.3.0. Can you verify if that fixes the problem so that we can update the dependency?

**Comment by yigitsoy** [comment]
Hi, thanks for your very quick support! I will test the new release today and let you know.

**Comment by yigitsoy** [comment]
Just tested and can verify that it works now! I had to update the dicom-pixeldata since it was fixed to 0.2.0. Please see https://github.com/Enet4/dicom-rs/pull/222

Again, thanks for the quick fix.
